content/en/docs/concepts/cluster-administration/addons.md

```diff
@@ -82,7 +82,7 @@ installation instructions. The list does not try to be exhaustive.
 * [Spiderpool](https://github.com/spidernet-io/spiderpool) is an underlay and RDMA
   networking solution for Kubernetes. Spiderpool is supported on bare metal, virtual machines,
   and public cloud environments.
-* [Weave Net](https://www.weave.works/docs/net/latest/kubernetes/kube-addon/)
+* [Weave Net](https://github.com/weaveworks/weave)
```

Comment:
```suggestion
```
Since Weave Works is shutting down, I propose to remove the line

Reply:
I have created:

*  https://github.com/kubernetes/website/pull/45445

Hopefully we can keep Weave Net in the guide 👍